Code/cmake/Modules/RDKitUtils.cmake

```diff
@@ -5,8 +5,13 @@ IF(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 ELSE(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   set(RDKit_VERSION "${RDKit_ABI}.${RDKit_Year}.${RDKit_Month}")
 ENDIF(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-set(RDKit_VERSION "${RDKit_VERSION}.${RDKit_Revision}${RDKit_RevisionModifier}")
-set(RDKit_RELEASENAME "${RDKit_Year}.${RDKit_Month}.${RDKit_Revision}${RDKit_RevisionModifier}")
+set(RDKit_RELEASENAME "${RDKit_Year}.${RDKit_Month}")
```

Comment:
This block overwrites some recent changes and should be reverted

Reply:
Oops, I missed that when I applied the diff from an older revision of trunk. I will revert it.